tntdb/src/mysql/connection.cpp

```diff
@@ -71,6 +71,9 @@ namespace tntdb
         throw std::runtime_error("cannot initalize mysql");
       initialized = true;
 
+      unsigned int timeout = 300;
```

Comment:
For posterity: As discussed in team, there are at least 3 types of timeout (connect, read, write) that can be tuned; our existing immediate problem was resolved by this one (read) - but we might need to come to others later.

Also at this time the timeout is effectively hardcoded to 5 minutes, as is reasonable for our practical use-case, however in later revisions we might want to mate this tunable somehow. Would speed up testing of the feature, in particular ;)